docs/components/gltf-model.md

```diff
@@ -0,0 +1,61 @@
+---
+title: gltf-model
+type: components
+layout: docs
+parent_section: components
+---
+
+[glTF][about-gltf] is an open project by Khronos providing a common, extensible format for 3D assets that is both efficient and highly interoperable with modern web technologies.
```

Comment:
Could describe give the advantages and purpose of glTF over other common formats (obj, collada). The .psd vs .jpg analogy is a good one